composer.json

```diff
@@ -48,20 +48,6 @@
             "Sentry\\Laravel\\Tests\\": "test/Sentry/"
         }
     },
-    "scripts": {
-        "tests": [
-            "vendor/bin/phpunit --verbose"
-        ],
-        "tests-travis": [
-            "XDEBUG_MODE=coverage vendor/bin/phpunit --verbose --configuration phpunit.xml --coverage-clover test/clover.xml"
-        ],
-        "tests-report": [
-            "XDEBUG_MODE=coverage vendor/bin/phpunit --verbose --configuration phpunit.xml --coverage-html test/html-report"
-        ],
-        "phpcs": [
-            "vendor/bin/php-cs-fixer fix --verbose --diff --dry-run"
-        ]
-    },
```

Comment:
I don't see any reason to remove these.

Reply:
yeah please don't, I love running `c phpcs` 👍 (`c` is alias to `composer` for me)

Reply:
`c` gang 🚀

Reply:
I removed the unused ones and synced them with the `Makefile`.